### PR TITLE
Issue 20.jadair.1

### PR DESCRIPTION
**Pull request by jeffadair**
# Pull request questions

## Which issue does this address

ISSUE-20

## Why was change needed

To enable pulling data from RabbitMQ as an alternative to Kafka

## What does change improve

Enable pulling data from RabbitMQ.

### CODE REVIEW

File: stream-loader.py [hunk 1]
```diff
@@ -588,6 +637,80 @@ def run(self):
         for thread in self.threads:
             thread.join()
 
+# -----------------------------------------------------------------------------
+# Class: KafkaProcess
```
Comment (docktermj) [inline stream-loader.py]: Should be RabbitMQProcess

File: stream-loader.py [hunk 2]
```diff
@@ -249,7 +292,8 @@ def get_parser():
     "127": "Monitor: {0}",
     "128": "Sleeping {0} seconds.",
     "129": "{0} is running.",
-    "130": "Sleeping infinitely.",
+    "130": "RabbitMQ channel closed by the broker. Shutting down thread {0}.",
+    "131": "Sleeping infinitely.",
```
Comment (docktermj) [inline stream-loader.py]: This is causing a down-stream mixup.

File: stream-loader.py (hunk 1 above)
Comment (jeffadair) [reply stream-loader.py]: Fixed

File: stream-loader.py (hunk 2 above)
Comment (jeffadair) [reply stream-loader.py]: Fixed! I swapped the downstream usages. Would have been easier to swap these. Too late now!

**Pull request merged**